src/routes/api/signUp/+server.js

```diff
@@ -1,33 +1,36 @@
-import { supabase, adminAuthClient } from '$lib/server/supabase.js';
-import { json, redirect } from '@sveltejs/kit';
+import { supabase } from "$lib/server/supabase.js";
+import { json, redirect } from "@sveltejs/kit";
 
 export async function POST({ request }) {
   try {
     const { email, password, name } = await request.json();
 
-    const { data: signUpData, error: signUpError } = await adminAuthClient.createUser({
-      email,
-      password,
-      email_confirm: true,
-      user_metadata: {
-        name,
-      },
-    });
+    const { data: signUpData, error: signUpError } = await supabase.auth.signUp(
+      {
+        email,
+        password,
+        options: {
+          emailRedirectTo: "http://localhost:5173/signIn",
+        },
+      }
+    );
 
     if (signUpError) {
       return json({ error: signUpError.message }, { status: 400 });
     }
 
     const { data: userData, error: profileError } = await supabase
-      .from('profiles')
+      .from("profile")
```

Comment:
I dont think this will work.

There are some fields in the profile table that are required that we are not passing through

Reply:
Hmmm... have checked and we have no required filed when creating the profile